package.json

```diff
@@ -25,7 +25,7 @@
     "lint"
   ],
   "dependencies": {
-    "@hmcts/look-and-feel": "1.13.4",
+    "@hmcts/look-and-feel": "^1.14.2",
```

Comment:
We've started removing the caret symbol `^` from our look-and-feel and one-per-page versions in order to prevent any updated version breaking our app.
Essentially a couple of weeks ago someone pushed a new version of a dependency which broke our app. So now we've decided to lock the version down.